### ISSUE TITLE
class 'faces' has no alt text for if the image is not visible 

### ISSUE THREAD

**Issue by Jen-Harris** [FAC-11/week1-dyrt#84]

**Comment by dangerdak** [comment]
Even decorative images should have empty alt text: https://www.abilitynet.org.uk/blog/five-golden-rules-compliant-alt-text
